index.html

```diff
@@ -53,5 +53,257 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+          <li class="btn-lang ">HTML</li>
+          <li class="btn-lang ">Bootstrap</li>
+          <li class="btn-lang ">Ruby</li>
+      </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+<section class="background-image">
+  <section class="image-content">
+      <div class="image-cont-holder">
+      <h1 class="image-h1">
+          Profesional Art Printing Data
+      </h1>
+      <p class="para">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+      </p>
+
+      <ul class="languages-2">
+        <li class="btn-lang ">HTML</li>
+        <li class="btn-lang ">Bootstrap</li>
+        <li class="btn-lang ">Ruby</li>
+    </ul>
+<div class="button-see-proj">
+<button class="see-project">
+  See Project
+</button>
+</div>
+  </div>
+  </section>
+</section>
+
+
+</section>
+</section>
+<section class="about-me">
+  <section class="about-me-page">
+    <div class="about-me-all">
+      <h1 class="about-me-h1">
+        About me
+      </h1>
+      <div class="about-me-para-div">
+      <p class="about-me-para">
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+
+    <div class="get-resume-div">
+    <button class="get-resume">
+      Get My Resume
+    </button>
+  </div>
+    </div>
+  </section>
+  </section>
+
+  
+  <section class="languages-framework-skills">
+    <ol class="general" id="general">
+      <li class="ol1 ">  
+        Language
+        <ul class="js gen">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </li>
+
+      <li class="ol2">
+        Frameworks
+        <ul class="framework gen">
+        <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </li>
+
+
+      
+        <li class="ol3">
+          Skills
+          <ul class="skill gen">
+          <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+      </li>
+    </ol>
+  </section>
+
+  <section class="my-interest">
+    <h1 class="my-interest-h1">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h1>
```

Comment:
- [ ] Kindly change the h1 to a lower hx tag because following semantic HTML on one page it should be only one h1 and it should be in the beginning. Keep rocking💪